pkg/process/model.go

```diff
@@ -64,25 +65,25 @@ type DepartmentSchedule struct {
 	ID         uint `gorm:"primaryKey"`
 	Department Department
 	Date       time.Time
-	HalfDay    HalfDayEnum `gorm:"default:0"`
+	HalfDay    HalfDayEnum `gorm:"check:name <> 'whole'"` // for department, the half day
```

Comment:
Dont use a check here, use a validator in our program rather than leave it to db would be a better way. It enable us to control and report certain error to the frontend.